lib/ddtrace/contrib/active_model_serializers/events/serialize.rb

```diff
@@ -0,0 +1,27 @@
+require 'ddtrace/contrib/active_model_serializers/event'
+
+module Datadog
+  module Contrib
+    module ActiveModelSerializers
+      module Events
+        # Defines instrumentation for !serialize.active_model_serializers event
+        module Serialize
+          include ActiveModelSerializers::Event
+
+          EVENT_NAME = '!serialize.active_model_serializers'.freeze
```

Comment:
This is the older < 0.10 event right? I would consider overriding the `supported?` method here and checking the version number, so it doesn't subscribe in 0.10+. See the ActiveRecord for [an example](https://github.com/DataDog/dd-trace-rb/blob/0.13-dev/lib/ddtrace/contrib/active_record/events/instantiation.rb#L17). Same likely applies to the `Render` event.